apps/condo/domains/user/integration/sberid/sync/syncUser.spec.js

```diff
@@ -0,0 +1,128 @@
+/**
+ * @jest-environment node
+ */
+
+const index = require('@app/condo/index')
+const { getItems } = require('@keystonejs/server-side-graphql-client')
+const faker = require('faker')
+const { v4: uuid } = require('uuid')
+
+const { setFakeClientMode } = require('@open-condo/keystone/test.utils')
+
+const { makeClientWithRegisteredOrganization } = require('@condo/domains/organization/utils/testSchema/Organization')
+const { SBER_ID_IDP_TYPE, RESIDENT } = require('@condo/domains/user/constants/common')
+const {
+    UserExternalIdentity: UserExternalIdentityApi,
+} = require('@condo/domains/user/utils/serverSchema')
+
+const { syncUser } = require('./syncUser')
+
+const { keystone } = index
+
+const mockUserInfo = (identityId, phone) => ({
+    id: identityId,
+    issuer: 'testIssuer',
+    email: faker.internet.email(),
+    phoneNumber: phone || faker.phone.phoneNumber('+792########'),
+})
+
+describe('syncUser from SBBOL', () => {
+    setFakeClientMode(index)
+
+    it('should create user', async () => {
+        // assemble
```

Comment:
This comment does not provides additional information. It's clear that before calling a test subject, we are assembling something
```suggestion
```